Stock_pre_model.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.metrics import mean_squared_error
+from keras.models import Sequential
+from keras.layers import LSTM, Dropout, Dense
```

Comment:
The use of 'MinMaxScaler' for feature scaling is not ideal for time series data. Consider using 'RobustScaler' or 'StandardScaler' which are more suitable for time series forecasting.